src/dom.js

```diff
@@ -605,3 +605,13 @@ export function tryFocus(element) {
     // IE <= 7 may throw exceptions when focusing on hidden items.
   }
 }
+
+/**
+ * Whether the given window is in an iframe or not.
+ * @param {!Window} win
+ * @return {boolean}
+ */
+export function isIframed(win) {
```

Comment:
I'm fine with this change, but let's discuss quickly an overall direction. I think eventually we will need to answer the question "What embedding environment is AMP document running in". The answer is not yet 100% clear but goes along these lines:
 - Standalone
 - Iframed *in a viewer*
 - Iframed *in a box*
 - Embedded in a webview
 - Embedded as a shadow doc
 - etc

So, I do see some sort of `Viewer.getEmbeddingType()` flag that could be used for some key decisions in the code.